Starting Garak job...
Running Garak scan with command: python3 -m garak --model_type huggingface --model_name gpt2 --probes dan.Ablation_Dan_11_0,dan.AntiDAN,dan.DAN_Jailbreak,dan.Dan_10_0,dan.Dan_11_0,dan.Dan_6_0,dan.Dan_6_2,dan.Dan_7_0,dan.Dan_8_0,dan.Dan_9_0,dan.DUDE,dan.STAN,dan.DanInTheWild,dan.DanInTheWildFull,dan.AutoDAN,dan.AutoDANCached,dan.ChatGPT_Image_Markdown,dan.ChatGPT_Developer_Mode_RANTI,dan.ChatGPT_Developer_Mode_v2 --report_prefix /Users/divyachitimalla/garak/dashboard/reports/b8a493c9-dff2-4f6e-b1b6-a1fcb4d84a6c
garak LLM vulnerability scanner v0.12.0.pre1 ( https://github.com/NVIDIA/garak ) at 2025-07-30T09:34:06.082416
ðŸ“œ logging to /Users/divyachitimalla/.local/share/garak/garak.log
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/Users/divyachitimalla/garak/garak/__main__.py", line 14, in <module>
    main()
Garak scan completed with exit code: 1
No reports generated
